data/json/effects_on_condition/npc_eocs/appearance_eocs.json

```
@@ -7,9 +7,10 @@
     "//3": "For ease of calculation, all math will be done with centimeters for hair growth.",
     "//4": "For reference starting point, a buzzcut is 0.32 centimeters long.",
     "//5": "1 inch is 2.54 centimeters.",
-    "recurrence": { "math": [ "(6480000 - ((u_val('health') * 180000) ) )" ] },
+    "//6": "Currently hair changes traits about every 20 days",
+    "recurrence": { "math": [ "150000" ] },
```

why comment says `every 20 days`, but recurrence happens roughly every 41 hours?

---

i assume it's a mistake, let's make it human readable
```suggestion
    "recurrence": { "math": [ "time('20 d')" ] },
```